Description: Adds the dependency on the Activation Framework to fix the build failure with Java 11.
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/istack-commons/runtime/pom.xml
+++ b/istack-commons/runtime/pom.xml
@@ -71,6 +71,11 @@
 
     <dependencies>
         <dependency>
+            <groupId>com.sun.activation</groupId>
+            <artifactId>javax.activation</artifactId>
+            <version>1.2.0</version>
+        </dependency>
+        <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
             <version>3.8.2</version>
